var/spack/repos/builtin/packages/py-pybind11/package.py

```diff
@@ -25,6 +27,8 @@ class PyPybind11(CMakePackage, PythonPackage):
     maintainers = ["ax3l"]
 
     version("master", branch="master")
+    version("2.10.0", sha256="eacf582fa8f696227988d08cfc46121770823839fe9e301a20fbce67e7cd70ec")
+    version("2.9.2", sha256="6bd528c4dbe2276635dc787b6b1f2e5316cf6b49ee3e150264e455a0d68d19c1")
```

Comment:
Were these new versions added as part of this PR? Seems outside scope if so.

Reply:
I can try to extract that, but these were added as part of fixing the `py-pybind11` build in pipelines.